packages/excalidraw/components/App.tsx

```diff
@@ -7900,6 +7901,31 @@ class App extends React.Component<AppProps, AppState> {
           !this.state.editingTextElement &&
           this.state.activeEmbeddable?.state !== "active"
         ) {
+
+          if(!this.state.editingLinearElement && !this.state.editingFrame && !this.state.resizingElement){
+
+            // this determines the threshold for dragging the element initially
+            const MOVEMENT_BUFFER = 4;
+  
+            // this calculates the buffer for the movement based on the zoom level
+            const ZOOM_RELETIVE_MOVEMENT_BUFFER = MOVEMENT_BUFFER / this.state.zoom.value;
+            
+            if(typeof(ZOOM_RELETIVE_MOVEMENT_BUFFER) === 'number'){
```

Comment:
Is this needed and safe enough?  I.e. typeof NaN is still 'number'

Reply:
I have made another commit and removed this check as it was unnecessary, and also made the changes you suggested.

What I found out is that when we compare NaN with any number be it zero or negetive (which should not be possible as long as the buffer or the zoom is not negative or the buffer is not zero) it always returns false on both >,< comparisons, so if at some point it happens to be NaN (which I think is impossible as long as the buffer and the zoom are positive numbers) , there would simply be no buffer. Please let me know if I am on the right track here.